bin/stub.rb.erb

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env ruby
+$LOAD_PATH.unshift('<%= rspec_bash_library_path_for_template %>')
+
+require 'rspec/bash'
+require 'pathname'
+require 'yaml'
+
+include Rspec::Bash
+
+command = File.basename(__FILE__)
+folder = File.dirname(__FILE__)
+
+call_log_path = Pathname.new(folder).join("#{command}_calls.yml")
```

Comment:
I'll do some research on it and see if I can remember why people were suggesting the other way. If it seems important then I'll make the change